include/mbgl/ios/MGLMapView.h

```diff
@@ -178,27 +165,20 @@
 
 /** @name Styling the Map */
 
-/** Sets the map style.
-*   @param styleJSON The map stylesheet as JSON text. */
-- (void)setStyleJSON:(NSString *)styleJSON;
-
-/** Returns the raw JSON style as a native dictionary object. */
-- (NSDictionary *)getRawStyle;
-
-/** Sets the raw JSON style as a native dictionary object with a transition animation.
-*   @param style The style JSON as a dictionary object. */
-- (void)setRawStyle:(NSDictionary *)style;
+/** Mapbox map ID of the style currently displayed in the receiver, or `nil` if the style does not have a map ID.
+ 
+    The style may lack a map ID if it is located at an HTTP, HTTPS, or asset: URL. Use -styleURL to get the URL in these cases.
+ */
+@property (nonatomic) IBInspectable NSString *mapID;
 
 /** Returns the names of the styles bundled with the library. */
 - (NSArray *)bundledStyleNames;
```

Comment:
Should we modify this to include full paths or style URLs to be passed, since we can't make direct use of the names returned here anymore?